plugin.json

```
@@ -15,7 +15,7 @@
     "keywords"          : ["plentymarkets", "plugins", "layout", "template", "ceres", "plentyshop", "webshop", "PlentyONE"],
     "type"              : "template",
     "require"           : {
-        "IO": "~5.0.73"
+        "IO": "~5.0.72"
```

This should be fixed by making a new io version
```suggestion
        "IO": "~5.0.73"
```